server/service-manager.js

```diff
@@ -0,0 +1,49 @@
+var debug = require('debug')('MeshModels.server.ServiceManager');
+
+function ServiceManager() {
+  debug('ServiceManager created');
+}
+
+function onServiceUpdate(service, callback) {
+  debug('onServiceUpdate(%j)', service);
+  setImmediate(callback);
+}
+ServiceManager.prototype.onServiceUpdate = onServiceUpdate;
+
+function onServiceDestroy(service, callback) {
+  debug('onServiceDestroy(%j)', service);
+  setImmediate(callback);
+}
+ServiceManager.prototype.onServiceDestroy = onServiceDestroy;
```

Comment:
LB aliases delete/destroy/remove etc. I prefer to call it destroy because in the scheduler case, its not just removing the model, but also cleaning up resources